build.gradle

```diff
@@ -16,7 +16,7 @@ repositories {
 }
 
 dependencies {
-    compile group: 'io.improbable', name: 'keanu', version: '0.0.15'
+    compile group: 'io.improbable', name: 'keanu', version: '0.0.17'
```

Comment:
We forgot 16!?!?

Reply:
the PR is still open....! It never got approved and I forgot to chase it up